### DISCUSSION TITLE
incorrect automatic repo language detection

### DISCUSSION THREAD

**Discussion by kklmn** [community/community#21950] [New to GitHub]
<p>Hello,</p>
<p>In my GitHub Profile/Overview I have a list of “Popular repositories”, the 1st one being mine and having a label “xBase” at the bottom, as shown below:<br>
<img src="https://user-images.githubusercontent.com/17322580/181067632-4cd2116a-40cd-4dbf-821b-f28eaf07ca84.png" alt="image.png" data-base62-sha1="v3D67mS0YxdHJn9wWW78zEzYqqW" width="465" height="345"></p>
<p>The repository has nothing to do with xBase. Where can I change it to “Python”? I’ve looped over the settings several times, it seems I’m missing something obvious, sorry for that.</p>
<p>Thanks for a hint in advance!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/kklmn">@kklmn</a> Thanks for being part of the GitHub Community Forum!</p>
<p>What you’re experiencing sounds similar to what was asked in <a href="https://github.community/t/disable-github-pages-in-project-language-statistics/10262">this other post</a>. A GitHub staff member responded there with more context about how languages are detected and how you can exclude certain files. I’d recommend checking out that post and continuing the conversation there.</p>

**Comment by kklmn** [comment]
<p>A great tip, it did help!</p>
<p>I;ve marked one directory as linguist-vendored in .gitattributes and “xBase” has gone.</p>
<p>Thanks!</p>

**Comment by nadiajoyce** [comment]
<p>Wonderful! I’m so glad it helped!</p>
